web/onos-gui/src/app/onos-topo/topodevice.service.ts

```diff
@@ -32,6 +34,7 @@ import { KeyValue } from '@angular/common';
 export class TopoDeviceService {
     deviceList: Map<string, Device>; // Expect <dev-id:dev-ver> as key
     topoDevicesSub: Subscription;
```

Comment:
you probably should create a `topoEntitySub : Subscription` too - it's there so we can have a handle on it when we need to close it

Reply:
okay